inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api.pyx

```diff
@@ -61,6 +61,12 @@ def get_version():
     return C.get_version().decode()
 
 
+def read_network(path_to_xml : str, path_to_bin : str):
```

Comment:
Is it temporary function? Do we have a plan to remove it in the future?
If yes, I propose to highlight it. Because python API already has read_network method for IECore.

Reply:
Im going to discuss how to make this method private with @akuporos when she is back from vacation.